Cmdr/BuiltInCommands/Admin/ban.lua

```diff
@@ -0,0 +1,22 @@
+return {
+	Name = "ban";
+	Description = "Bans a player or set of players.";
+	Group = "DefaultAdmin";
+	Args = {
+		{
+			Type = "playerIds";
+			Name = "players";
+			Description = "The players to ban.";
+		},
+		{
+			Type = "duration";
+			Name = "duration";
+			Description = "How long the ban should last. A negative value means a permanent ban.";

```

Comment:
I agree with this. I believe we could just keep the existing command and use a prefixed union type. Though, with a type of `"duration ! string"` and a value of just `!`, the type validator for primitive strings would flag it, as it's considered a nil value.

Maybe this could be resolved by requiring the user to use `!permanent` as the value?